app/src/main/java/org/schabi/newpipe/ReCaptchaActivity.java

```diff
@@ -155,6 +150,16 @@ public boolean onOptionsItemSelected(final MenuItem item) {
         }
     }
 
+    private String getUrlFromIntent() {
+        final String url = getIntent().getStringExtra(RECAPTCHA_URL_EXTRA);
+        if (url == null || url.isEmpty()) {
+            return YT_URL; // YouTube is the most likely service to have thrown a recaptcha
+        } else {
+            // remove "pbj=1" parameter from YouYube urls, as it makes the page JSON and not HTML
+            return url.replace("&pbj=1", "").replace("?pbj=1&", "").replace("?pbj=1", "");
```

Comment:
This looks like it would break urls like `www.host.com/?pbj=1&otherparam=2` by turning it to `www.host.com/otherparam=2`

Reply:
Oh, I accidentally left an additional ? in there